initrd/etc/functions

```diff
@@ -81,6 +81,36 @@ confirm_totp()
 	echo
 }
 
+reseal_tpm_disk_decryption_key()
+{
+	TRACE "Under /etc/functions:reseal_tpm_disk_decryption_key"
+	#Can be called only on resealing, not on tpm reset:
+	#otherise primary handle and rollback would change.
+	if ! grep -q /boot /proc/mounts ; then
+		mount -o ro /boot \
+		|| recovery "Unable to mount /boot"
+	fi
+
+	if [ -s /boot/kexec_key_devices.txt ] || [ -s /boot/kexec_key_lvm.txt ]; then
+		warn "A TPM disk decryption key previoulsy sealed is now invalid since firmware measurements cannot be unsealed" 
+		warn "Renewing LUKS Disk Unlock Key to be unsealed by TPM Disk Unlock Key passphrase"
+		while ! kexec-seal-key /boot; do
+			warn "Recovery disk encryption key passphrase invalid. Try again!"
+		done
+		warn "LUKS header hash changed under of /boot/kexec_luks_hdr_hash.txt" 
```

Comment:
Grammar: "under of", I think delete "of" and leave "...hash changed under /boot/..."

Reply:
Fixed in 8809588a :+1: